db/seeds.rb

```diff
@@ -1,7 +1,13 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+DynamicFieldCategory.destroy_all
+DynamicFieldGroup.destroy_all
+DynamicField.destroy_all
+
+user = User.find(1);
+
+DynamicFieldCategory.create!({id: 1, display_label: "Descriptive Metadata", sort_order: 1})
```

Comment:
I think the curly braces here are redundant. Though, I am curious as to why rubocop did not catch that.